sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jdbc/nativejdbc/implementation/JdbcPluginPropertiesUtils.java

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.autoconfigure.jdbc.nativejdbc.implementation;
+
+import com.azure.spring.cloud.core.implementation.properties.PropertyMapper;
+import com.mysql.cj.conf.PropertySet;
+
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Store the constants for customized Azure properties with JDBC.
+ */
+public class JdbcPluginPropertiesUtils {
```

Comment:
isn't this class duplicated by `AzureJdbcPropertiesUtils`?

Reply:
It's something like that,
the differences here is `JdbcPluginPropertiesUtils ` depends on nothing about spring as expected, while AzureJdbcPropertiesUtils depends on `spring-cloud-azure-core` .

Reply:
this is a bad design and brings lots of duplication.
there are tons of design patterns for this kind of requirement